### PR TITLE
Refresh the OS Druid web console screenshots

### PR DESCRIPTION
**Pull request by demo-kratia**
Images update in Druid web console page.

This PR has:

- [x] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] a release note entry in the PR description.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

**Comment by clintropolis** [comment]
sorry to be the bearer of bad news, but some of these will need refreshed again from the ui changes of #14395 which splits the ingestion view into separate supervisor and task views

File: docs/operations/web-console.md [hunk 1]
```diff
@@ -127,50 +127,54 @@ To display a timeline of segments, toggle the option for **Show segment timeline
...
+You can group the nodes by **Type** or by **Tier** to get meaningful summary statistics. 
 
-![servers](../assets/web-console-10-servers.png)
+![Services](../assets/web-console-10-servers.png)
 
 
 ## Lookups
 
-Access the **Lookups** view from the **Lookups** card in the home view or by clicking on the gear icon in the upper right corner.
+Access the **Lookups** view from the **Lookups** card in the home view or by clicking on the ellipsis icon in the top-level navigation.
```
Comment (ektravel) [inline docs/operations/web-console.md:177]:
```suggestion
Access the **Lookups** view from the **Lookups** card in the home view or by clicking the ellipsis icon in the top-level navigation.
```

File: docs/operations/web-console.md [hunk 2]
```diff
@@ -127,50 +127,54 @@ To display a timeline of segments, toggle the option for **Show segment timeline
...
-## Supervisors and tasks
+## Supervisors
 
 From this view, you can check the status of existing supervisors as well as suspend, resume, and reset them.
-The supervisor oversees the state of the indexing tasks to coordinate handoffs, manage failures, and ensure that the scalability and replication requirements are maintained.
+The supervisor oversees the state of the indexing tasks to coordinate handoffs, manage failures, and ensure that the scalability and replication requirements are maintained. Submit a supervisor manually by clicking the ellipsis icon and selecting **Submit JSON supervisor**.
+
+![Supervisors](../assets/web-console-07-supervisors.png)
+
+Click on the magnifying glass for any supervisor to see detailed reports of its progress.
```
Comment (ektravel) [inline docs/operations/web-console.md:151]:
```suggestion
Click the magnifying glass icon for any supervisor to see detailed reports of its progress.
```

File: docs/operations/web-console.md [hunk 3]
```diff
@@ -127,50 +127,54 @@ To display a timeline of segments, toggle the option for **Show segment timeline
...
 Submit a task manually by clicking the ellipsis icon and selecting **Submit JSON task**.
 
-![supervisors](../assets/web-console-07-supervisors.png)
-
-Click on the magnifying glass for any supervisor to see detailed reports of its progress.
-
-![supervisor-status](../assets/web-console-08-supervisor-status.png)
+![Tasks](../assets/web-console-0.7-tasks.png)
 
 Click on the magnifying glass for any task to see more detail about it.
```
Comment (ektravel) [inline docs/operations/web-console.md:163]:
```suggestion
Click the magnifying glass icon for any task to see more detail about it.
```

**Review by ektravel — COMMENTED** [review]
A few nits. Looks good otherwise.

**Review by ektravel — APPROVED** [review]
LGTM

File: docs/operations/web-console.md [hunk 4]
```diff
@@ -127,50 +127,54 @@ To display a timeline of segments, toggle the option for **Show segment timeline
...
 
-![segments](../assets/web-console-06-segments.png)
+![Segments](../assets/web-console-06-segments.png)
 
-## Supervisors and tasks
+## Supervisors
 
 From this view, you can check the status of existing supervisors as well as suspend, resume, and reset them.
-The supervisor oversees the state of the indexing tasks to coordinate handoffs, manage failures, and ensure that the scalability and replication requirements are maintained.
+The supervisor oversees the state of the indexing tasks to coordinate handoffs, manage failures, and ensure that the scalability and replication requirements are maintained. Submit a supervisor manually by clicking the ellipsis icon and selecting **Submit JSON supervisor**.
```
Comment (vtlim) [inline docs/operations/web-console.md:147]:
```suggestion
The supervisor oversees the state of the indexing tasks to coordinate handoffs, manage failures, and ensure that the scalability and replication requirements are maintained. Submit a supervisor spec manually by clicking the ellipsis icon and selecting **Submit JSON supervisor**.
```

File: docs/assets/web-console-13-lookups.png
Comment (vtlim) [inline docs/assets/web-console-13-lookups.png:1]: I think we want to call it "ellipsis icon" instead of configuration icon with the moved menu location

**Review by vtlim — APPROVED** [review]

**Pull request merged**